elixir-mode.el

```diff
@@ -405,6 +405,14 @@ is used to limit the scan."
                      (one-or-more "\n")))
      1 font-lock-variable-name-face)
 
+    ;; Gray out variables starting with "_"
+    (,(elixir-rx symbol-start
+                 (group (and "_"
```

Comment:
I thought about extracting the contents of this `group` to a variable like it is done with `sigils` and `identifiers`, but I couldn't find a descriptive name, so I've left the definition inlined.

What do you guys think?